### DISCUSSION TITLE
Dark mode theme version?

### DISCUSSION THREAD

**Discussion by mordadam** [majidh1/JalaliDatePicker#121] [Question] [answered]
How can I use the dark theme version of calendar?
You uploaded an image in the description, but I don't know how to use it.

Thanks!

----------------------------------------------------
چگونه می توانم از نسخه تم تیره تقویم استفاده کنم؟
شما تصویر تم تیره را در توضیحات آپلود کردید، اما من نمی دانم چگونه از آن استفاده کنم.

متشکرم!

![sample5](https://github.com/user-attachments/assets/475be6d4-eb19-4915-972d-3b6f842b8ac1)

**Comment by majidh1** [comment, marked as answer]
Hello! 🌟
you can refer to the solution provided in https://github.com/majidh1/JalaliDatePicker/issues/6#issuecomment-860003823
